collector/mssql.go

```diff
@@ -1848,465 +1879,461 @@ func (c *MSSQLCollector) Collect(ctx *ScrapeContext, ch chan<- prometheus.Metric
 // - Win32_PerfRawData_MSSQLSERVER_SQLServerAccessMethods
 //   https://docs.microsoft.com/en-us/sql/relational-databases/performance-monitor/sql-server-access-methods-object
 type win32PerfRawDataSQLServerAccessMethods struct {
```

Comment:
With the WMI lib, we had to match type names to class names, but this is no longer the case, so we can give these types names without all the WMI-isms. So something like `mssqlAccessMethods` in this case, maybe? Same pattern for all the other things mentioning perfraw etc.

Reply:
Please see commit 2d015fa3b7aeb9a1f7d57b243065ba44a9fc8f72 containing renamed structs.